### DISCUSSION TITLE
Dynamic permissions

### DISCUSSION THREAD

**Discussion by hanifhefaz** [HassanZahirnia/laravel-package-ocean#49] [Ideas]
How to write permissions that is automatically checked in blade?

**Comment by HassanZahirnia** [comment]
Hey, if you have any general Laravel questions there are many places that you could do that 😊. Will close this as it's off-topic

**Reply by hanifhefaz** [reply]
It is for sure off-topic. I thought of the ideas to be converted to packages. Thank you for your time.
